src/runtime/README_webgpu.md

```diff
@@ -0,0 +1,100 @@
+# WebGPU support for Halide
```

Comment:
This README should probably me moved to the same location as all the other READMEs